spec/sidekiq/batch_spec.rb

```diff
@@ -227,10 +227,11 @@ def was_performed; end
 
   describe '#enqueue_callbacks' do
     let(:callback) { double('callback') }
-    let(:event) { :complete }
+    let(:event) { "complete" }
 
     context 'on :success' do
-      let(:event) { :success }
+      let(:event) { "success" }
```

Comment:
Is this needed in the spec, since it's converted to string in the `event_name` so maybe it's even better to have it as symbol for the sake of test.

Reply:
:ok_hand: